tests/index.js

```diff
@@ -106,6 +106,8 @@ describe('Thorn', () => {
                         name: 'TestRecord1',
                         testField1: 'TestField1data',
                         testField2: 'TestField2data',
+                        testField3: false,
```

Comment:
I think I've discovered the reason: the fixtures object is somehow being modified as a reference. If you use `_.clone()` on it, you'll discover that the test fails once you revert the primary change.